modules/base/shaders/pointcloud/pointcloud_fs.glsl

```diff
@@ -85,21 +92,52 @@ Fragment getFragment() {
   }
 
   // Moving the origin to the center and calculating the length
-  float lengthFromCenter = length((texCoord - vec2(0.5)) * 2.0);
-  if (!hasSpriteTexture && (lengthFromCenter > 1.0)) {
+  vec2 centeredTexCoords = (texCoord - vec2(0.5)) * 2.0;
+  float lengthFromCenter = length(centeredTexCoords);
+
+  bool shouldBeRound = (!hasSpriteTexture && !enableOutline) ||
+    (enableOutline && outlineStyle == OutlineStyleRound);
+
+  if (shouldBeRound && (lengthFromCenter > 1.0)) {
```

Comment:
I am not sure but it feels like this should be reversed? If the distance to the center is **smaller** than 1 it should be discarded. Or am I wrong?

Reply:
We want to discard all pixels outside the radius == 1, to get a circular point.  So, yes, you're wrong :)